patches/entropy-0.4.cabal

```diff
@@ -0,0 +1,54 @@
+name:           entropy
+version:        0.4
+x-revision: 1
+description:    A platform independent method to obtain cryptographically strong entropy
+                (RDRAND, urandom, CryptAPI, and patches welcome)
+                Users looking for cryptographically strong (number-theoretically
+                sound) PRNGs should see the 'DRBG' package too.
+synopsis:       A platform independent entropy source
+license:        BSD3
+license-file:   LICENSE
+copyright:      Thomas DuBuisson <thomas.dubuisson@gmail.com>
+author:         Thomas DuBuisson <thomas.dubuisson@gmail.com>
+maintainer:     Thomas DuBuisson <thomas.dubuisson@gmail.com>
+category:       Data, Cryptography
+homepage:       https://github.com/TomMD/entropy
+bug-reports:    https://github.com/TomMD/entropy/issues
+stability:      stable
+-- build-type:  Simple
+-- ^^ Used for HaLVM
+build-type:     Custom

```

Comment:
`etlas` doesn't support `build-type: Custom` so you should probably switch it to `build-type: Simple`.